bond/package.xml

```diff
@@ -1,31 +1,34 @@
 <?xml version="1.0"?>
-<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
-<package format="2">
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
   <name>bond</name>
-  <version>1.8.3</version>
+  <version>1.8.4</version>
   <description>
     A bond allows two processes, A and B, to know when the other has
     terminated, either cleanly or by crashing.  The bond remains
     connected until it is either broken explicitly or until a
     heartbeat times out.
   </description>
   <maintainer email="michael@openrobotics.org">Michael Carroll</maintainer>
-  <license>BSD</license>
-
+  <license>Apache License 2.0</license>
```

Comment:
Since the original repo was BSD, we need to stay with that in absence of approval of the original authors.

Reply:
@mjcarroll 
I have restored the license information, I have kept previous license and added new one but kept as commented, If needed it can be removed.